op-guide/binary-testing-deployment.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Testing Deployment from Binary Tarball
+summary: Use the binary to deploy a TiDB cluster.
+category: operations
+---
+
+# Testing Deployment from Binary Tarball
+
+This guide provides installation instructions for all TiDB components across multiple nodes for testing purposes. It does not match the recommended usage for production systems.
+
+See also [local deployment](../op-guide/binary-local-deployment.md) and [production enviroment](../op-guide/binary-deployment.md) deployment.
+
+## Prepare
+
+Before you start, see [TiDB architecture](/overview.md#tidb-architecture) and [Software and Hardware Requirements](/op-guide/recommendation.md). Make sure the following requirements are satisfied:
+
+### Operating system
+
+For the operating system, it is recommended to use RHEL/CentOS 7.3 or higher. The following additional requirements are recommended:
+
+| Configuration | Description |
+| :-- | :-------------------- |
+| Supported Platform | RHEL/CentOS 7.3+ ([more details](/op-guide/recommendation.md)) |
+| File System  |  ext4 is recommended |
+| Swap Space  |  Should be disabled  |
+| Disk Block Size  |  Set the system disk `Block` size to `4096` |
+
+### Network and firewall
+
+| Configuration | Description |
+| :-- | :------------------- |
+| Firewall/Port | Check whether the ports required by TiDB are accessible between the nodes |
+
+### Operating system parameters
+
+| Configuration | Description |
```

Comment:
Might be good to show folks how to accomplish all of these topics instead of just some. (eg Transparent Hugepages)

Reply:
I agree.  I will leave it for future work though :-)